mist/worker/src/main/scala/io/hydrosphere/mist/worker/runners/python/PythonRunner.scala

```diff
@@ -57,11 +59,9 @@ class PythonRunner(jobFile: File) extends JobRunner with Logger {
         gatewayServer.shutdown()
       }
 
-      Right(entryPoint.dataWrapper.get)
+      Future.successful(entryPoint.dataWrapper.get)
```

Comment:
can't understand why it returns `Future`. I see only `succcesful` and `failure` usages.

Reply:
I don't know why I used it here. I'll revert it to use `Either`.